web/pdf_page_view.js

```diff
@@ -44,41 +44,29 @@ var TEXT_LAYER_RENDER_DELAY = 200; // ms
  */
 
 /**
- * @class
  * @implements {IRenderableView}
  */
-var PDFPageView = (function PDFPageViewClosure() {
+class PDFPageView {
   /**
-   * @constructs PDFPageView
    * @param {PDFPageViewOptions} options
    */
-  function PDFPageView(options) {
-    var container = options.container;
-    var id = options.id;
-    var scale = options.scale;
-    var defaultViewport = options.defaultViewport;
-    var renderingQueue = options.renderingQueue;
-    var textLayerFactory = options.textLayerFactory;
-    var annotationLayerFactory = options.annotationLayerFactory;
-    var enhanceTextSelection = options.enhanceTextSelection || false;
-    var renderInteractiveForms = options.renderInteractiveForms || false;
-
-    this.id = id;
-    this.renderingId = 'page' + id;
-    this.pageLabel = null;
+  constructor(options) {
```

Comment:
I chose to not do that because it would indeed be too verbose. I usually keep a limit at five or six parameters: if we have more, than such an expansion does not provide much more insight in my opinion.